tests/lib.rs

```diff
@@ -419,3 +419,7 @@ fn test_group_layers() {
         layer_tile_3.properties.get("key")
     );
 }
+
+mod libtest {
```

Comment:
What's that for?

Reply:
Oops. When making doctests I test them out out of the comment so it's easier to work with them. This is a byproduct, forgot to remove it.